src/task.ts

```diff
@@ -42,6 +42,7 @@ export type Task<A extends string = never> = {
   limit?: Limit;
   outfit?: OutfitSpec | Outfit | (() => OutfitSpec | Outfit);
   combat?: CombatStrategy<A>;
+  outfitOptions?: Partial<MaximizeOptions> | (() => Partial<MaximizeOptions>);
```

Comment:
Maybe we could move these `outfitOptions` into a `options?: Partial<MaximizeOptions>` on the OutfitSpec?

Reply:
I thought about that; my concern is twofold:
1. I think it can be counterintuitive to have the same information appear in two separate places on the same object, which I suppose is also an issue here. But between forceEquips and preventEquips and other fields on the MaximizeOptions, I think letting players set both things on a specific spec feels dangerous. That being said, I suppose you could say the exact same thing about setting them on the task.
2. I kind of added this as an afterthought upon discovering that outfit.dress takes a second argument that engine.dress will never use

Reply:
Honestly, I would be ok with removing the argument in outfit.dress entirely and moving the Partial<MaximizeOptions> entirely into the outfit+outfit spec. The only reason outfit.dress takes an argument is exactly to force an inventory refresh if the maximizer fails in [loopgyou](https://github.com/Kasekopf/loop-casual/blob/004e1c64bb9a994018f39ffed92464f729cbd830/src/engine/engine.ts#L565). I'm pretty sure loopgyou and loopcasual are the only scripts using the argument in outfit.dress.

Reply:
To your point 1, I agree that the same thing could be said for either putting them on the task or on the spec. I think putting them on the spec is easier to hide, with maybe a comment that they are not recommended except for advanced use cases.

Reply:
I'll move outfit options onto the spec & the outfit class. It'll be slightly annoying to copy all of its iterable elements but not _that_ annoying

Reply:
This reference to MaximizeOptions is still here.